tests/test_all.py

```diff
@@ -122,6 +135,23 @@ def test_rename_module():
         assert pathlib.Path("fruity.py").exists()
 
 
+def test_rename_module__dry():
+    runner = click.testing.CliRunner()
+    with runner.isolated_filesystem():
+        veg, fruit = make_veg_fruit_pyfiles()
+
+        with CaptureOutput() as output:
+            run(runner, "rename", "--dry", fruit, "fruity")
+            assert output.getvalue()
+
+        with CaptureOutput() as output:
+            run(runner, "rename", "--dry", f"{fruit}::cherries", "berries")
+            assert output.getvalue()
```

Comment:
Not exactly happy with this very bare bones assertion but should suffice

Reply:
Have added a check for `Renaming`, seems slightly better now